### PR TITLE
CP-11100: [CHORE] Update and sync policies for latest AWS billing changes

### PR DESCRIPTION
**Pull request by Tankanow**
## Description of the change

> [CHORE] Update and sync policies for latest AWS billing changes

## Type of change
- [ ] Bug fix
- [ ] New feature

### Development
- [ ] All changed code has 80% unit test coverage
- [x] All changed code has been automatically (smoke test or otherwise) or manually verified in `alfa` (or with a cross namespace setup, e.g. developer namespace for this feature, pointing at shared `alfa` resources)

### Code review
- [x] This pull request has a title that includes the ticket # and a short useful summary, e.g. `CP-4051: Create TEMPLATE Feature Repo`.

### CODE REVIEW

**Comment by Tankanow** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #71** <a href="https://app.graphite.dev/github/pr/Cloudzero/provision-account/71" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/Cloudzero/provision-account/71?utm_source=stack-comment).

File: policies/master_payer.json [hunk 1]
```diff
@@ -29,16 +29,28 @@
     },
     {
       "Action": [
-        "aws-portal:View*",
+        "account:GetAccountInformation",
```
Comment (Tankanow) [inline policies/master_payer.json:32]: NOTE: @muellerk22 and @silvexis, I made the changed described in https://cloudzero.atlassian.net/browse/CP-11100.

File: services/account_type/master_payer.yaml [hunk 2]
```diff
@@ -274,58 +276,97 @@ Resources:
                   - 'arn:aws:s3:::${BucketName}-${Id}/*'
                   - { BucketName: !FindInMap [ Defaults, Cur, BucketName ],
                       Id: !Select [ 2, !Split [ '/', !Ref 'AWS::StackId' ] ] }
-          - Sid: CZCostMonitoring20191111
+          - Sid: CZProtect20190912
```
Comment (Tankanow) [inline services/account_type/master_payer.yaml:279]: NOTE: @muellerk22 and @silvexis, also I synced the JSON policies above into our automated CloudFormation policies here b/c the JSON policies are newer. Please review theses changes.

File: services/account_type/master_payer.yaml [hunk 3]
```diff
@@ -274,58 +280,97 @@ Resources:
...
+            Effect: Deny
+            Action:
+              - aws-portal:*PaymentMethods
+              - aws-portal:*Account
+              - ec2:GetPasswordData
+              - ec2:GetConsoleOutput
+              - ec2:GetConsoleScreenshot
+              - ec2:GetLaunchTemplateData
+              - glue:GetConnection
```
Comment (Tankanow) [inline services/account_type/master_payer.yaml:292]: QUESTION: @silvexis, these are copied as is from the existing json policies. Do we need to update the Deny list too? Do we need to replace `aws-portal` with some new Deny Actions?

Comment (silvexis) [reply services/account_type/master_payer.yaml:292]: ah, you know what, lets just delete the whole deny section from all our policies. They are unnecessary and just confuse people anyway...

File: policies/master_payer.json [hunk 4]
```diff
@@ -29,16 +29,28 @@
...
-        "organizations:List*"
+        "organizations:List*",
+        "payments:Get*",
+        "payments:List*",
+        "pricing:*",
+        "tax:Get*",
+        "tax:List*"
       ],
       "Resource": "*",
       "Effect": "Allow",
```
Comment (muellerk22) [inline policies/master_payer.json:56]: Should the Policies "Sid" be updated to reflect the change from CZCostMonitoring20210423 to CZCostMonitoring20230118?

File: policies/resource_owner.json [hunk 5]
```diff
@@ -17,16 +17,28 @@
...
-        "organizations:List*"
+        "organizations:List*",
+        "payments:Get*",
+        "payments:List*",
+        "pricing:*",
+        "tax:Get*",
+        "tax:List*"
       ],
       "Resource": "*",
       "Effect": "Allow",
```
Comment (muellerk22) [inline policies/resource_owner.json:44]: Should the Sid be updated to reflect the change?

File: services/account_type/master_payer.yaml (hunk 3 above)
Comment (Tankanow) [reply services/account_type/master_payer.yaml:292]: OK

File: policies/master_payer.json (hunk 4 above)
Comment (silvexis) [reply policies/master_payer.json:56]: yes

File: policies/resource_owner.json (hunk 5 above)
Comment (silvexis) [reply policies/resource_owner.json:44]: yes

**Review by muellerk22 — APPROVED** [review]
Looks good to me.

**Review by silvexis — APPROVED** [review]

**Pull request merged**